testsuite/model/src/test/java/org/keycloak/testsuite/model/session/OfflineSessionPersistenceTest.java

```diff
@@ -189,7 +218,27 @@ public void testPersistenceMultipleNodesClientSessionsAtRandomNode() throws Inte
                 int oid = index % offlineSessionIds.size();
                 String offlineSessionId = offlineSessionIds.get(oid);
                 int cid = index % clientIds.size();
-                String clientSessionId = createOfflineClientSession(offlineSessionId, clientIds.get(cid));
+                String clientSessionId;
+                while (true) {
+                    try {
+                        clientSessionId = createOfflineClientSession(offlineSessionId, clientIds.get(cid));
+                        break;
+                    } catch (RuntimeException ex) {
+                        // invocation can fail when remote cache is stopping, this is actually part of this test:
+                        // "ISPN000217: Received exception from node-8, see cause for remote stack trace
+                        // IllegalLifecycleStateException: ISPN000324: Cache 'clientSessions' is in 'STOPPING' state and this is an invocation not belonging to an
+                        // on-going transaction, so it does not accept new invocations."
+                        if (ex.getMessage().contains("ISPN000217")) {
```

Comment:
Should this target `ISPN000324` rather than `ISPN000217`? It seems that the current code could mask even valid exceptions.

Reply:
I changed the code to look in the nested exception with the `ISPN000324`. Please re-review.